### DISCUSSION TITLE
Fuzzy max_expansion and prefixlength support

### DISCUSSION THREAD

**Discussion by anshulgoel27** [quickwit-oss/quickwit#5449] [General]
Does quickwit support fuzzy query feature similar to what lucene offers like the max_exapansion and prefixLength?

https://lucene.apache.org/core/7_1_0/core/org/apache/lucene/search/FuzzyQuery.html#:~:text=Learn%20how%20to%20use%20FuzzyQuery%20to%20perform%20fuzzy%20search%20with

**Comment by PSeitz** [comment]
Currently we only have prefix queries:
https://quickwit.io/docs/reference/es_compatible_api#match_phrase_prefix
